### DISCUSSION TITLE
"Client Session Idle" and "Client Session Max" in Keycloak 19 admin console

### DISCUSSION THREAD

**Discussion by dlmorais-13** [keycloak/keycloak#14225] [Archive]
Hi,

I started trying Keycloak 19 in the last few days and I can't find on new admin console the option to set session timeout on client level anymore. Those options existed on previous admin console (image below).

![image](https://user-images.githubusercontent.com/17011099/188646370-695c5715-a0bf-471f-b9cc-c463ee136b5f.png)

Where are these options on new admin console? Where they deprecated and removed?

**Comment by alexlanz** [comment]
We are having the same issue. We cannot find these settings anymore. Are there any updates on this?

**Reply by souravs17031999** [reply]
It seems they are now in realm settings - 
![client](https://user-images.githubusercontent.com/33771969/200853681-6a17b9cf-2a7a-49d8-a1ad-07306a4e0027.png)

and in clients, you can configure access token lifespan 
![clients](https://user-images.githubusercontent.com/33771969/200854084-bdbb621a-840a-4ee7-866b-c738fe70a32e.png)

**Reply by alexlanz** [reply]
Thank you for your reply. In deed, now in version 20 it is back.
